sopel/config/types.py

```diff
@@ -236,17 +266,22 @@ def configure(self, prompt, default, parent, section_name):
             prompt = prompt[0]
 
         if default is not NO_DEFAULT:
-            default = ','.join(default)
-            prompt = '{} [{}]'.format(prompt, default)
+            default_prompt = ','.join(['"{}"'.format(item) for item in  default])
```

Comment:
This is what's failing Travis builds rn.
```suggestion
            default_prompt = ','.join(['"{}"'.format(item) for item in default])
```

Reply:
Aww snap. I fixed it after my first push, but I `commit`'ed again without `add`'ing :cry:

Reply:
Lol, we've all done that. 😂

Also, TIL OctoDroid collapses whitespace in comments (it doesn't show the diff in my suggested change properly; both lines look identical).